sqrl.go

```diff
@@ -63,8 +63,6 @@ type QueryRowerContext interface {
 type BaseRunner interface {
 	Execer
 	ExecerContext
-	Queryer
```

Comment:
The role of the `BaseRunner` was to determine minimal functionality (be able to `Query` and `Exec`) of some DB connection in compile-time. Now `Query*` methods are gone and the only thing we require from DB connection is existence of `Exec*` methods. That renders BaseRunner less useful, so it seems like it should be removed completely and `interface{}` should be used instead of it.
But that means no compile-time checks, only runtime checks in `wrapRunner` function. Which means "rebuilding lots of stuff". I don't think we should go this way.